TheFourFifths/consus
# Add attributes for creating new models to post.

### Summary

I missed some things in the last iteration of this task. Needed to add the updated vars to creating a new model.

### Checklist


Finally, is it ready to be reviewed and merged: yes 🍏 

### How to Review

1. Checkout this branch
2. Run the server
3. Checkout consus-client branch=add-model-tffsomething
4. Start client
5. Click view all models
6. Verify all data shows as defined in the REST.md of this repo

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-91](https://msoese.atlassian.net/browse/TFF-91)
- [Add models client](https://github.com/TheFourFifths/consus-client/pull/20)
